src/webpubsub/azext_webpubsub/client.py

```diff
@@ -0,0 +1,112 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+import asyncio
+import sys
+import threading
+import json
+import websockets
+from .vendored_sdks.azure_messaging_webpubsubservice import (
+    build_authentication_token
+)
+
+
+async def connect(url):
+    async with websockets.connect(url, subprotocols=['json.webpubsub.azure.v1']) as ws:
+        comment = """
+----------Usage-----------
+joingroup <group-name>              : Join the connection to group
+leavegroup <group-name>             : Leave the connection from group
+sendtogroup <group-name> <message>  : Send message to group
+event <event-name> <message>        : Send event to event handler
+--------------------------
+        """
+        print(comment)
```

Comment:
I recommend to print comments to stderr

Reply:
`print` here is not an error

Reply:
I see, CLI usually print help message and error message in stderr stream.

Reply:
What's the recommend way to print message to stderr in CLI, `print('message', file=sys.stderr)`?

Reply:
@jiasli, do you have any recommendation?

Reply:
`stderr` is used for error and **diagnostic** messages: https://www.cplusplus.com/reference/cstdio/stderr/

But for interactive commands like this one, using `stderr` and `stdout` makes little difference.